@ecomplus/storefront-template/content/widgets/martan.json

```
@@ -0,0 +1,21 @@
+{
+  "pkg": "@ecomplus/widget-martan",
+  "active": true,
```

```suggestion
  "active": false,
```
Widget precisa inicializar desativado

---

esse arquivo pode ser ignorado